### PR TITLE
add public endpoints

### PR DESCRIPTION
**Pull request by fkenji**
Add documentation for public endpoints

/cc @azenMatt @kendramoroz @zhangchiqing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -312,7 +317,196 @@ NOTE: If an issue is from a different repository than the epic it belongs to, th
...
+```
+
+Please note: `repo_id` is the ID of the repository, not its full name. For example, the ID of the `ZenHubIO/support` repository is `13550592`.
+To find out the ID of your repository, use [GitHub's API](https://developer.github.com/v3/repos/#get).
+
+`issue_number` is the issue that we want to set the estimate for.
+
+#### Request Body
+
+```
```
Comment (zhangchiqing) [inline README.md]: use "```json" to format json code in Markdown

File: README.md [hunk 2]
```diff
@@ -312,7 +317,196 @@ NOTE: If an issue is from a different repository than the epic it belongs to, th
...
+
+Please note: `repo_id` is the ID of the repository, not its full name. For example, the ID of the `ZenHubIO/support` repository is `13550592`.
+To find out the ID of your repository, use [GitHub's API](https://developer.github.com/v3/repos/#get).
+
+`issue_number` refers to the issue that we want to convert to an epic.
+
+#### Request body
+
+```json
+{
```
Comment (zhangchiqing) [inline README.md]: Could you format the json with maybe online json formatter?

**Comment by zhangchiqing** [comment]
Looks good :shipit:

File: README.md [hunk 3]
```diff
@@ -11,6 +11,11 @@ This document outlines the setup and usage of ZenHub's API: a RESTful API with J
    repository](#get-the-zenhub-board-data-for-a-repository)
   - [Get Epics for a repository](#get-epics-for-a-repository)
   - [Get Epic data](#get-epic-data)
+  - [Move issue between pipelines](#move-issue-between-pipelines)
```
Comment (pnavarrc) [inline README.md]: The use of upper case and lower case should be consistent, also some articles are missing.

**Pull request merged**